src/bindings/python/tests/test_runtime/test_properties.py

```diff
@@ -28,8 +28,13 @@ def test_property_ro():
     assert "available_devices(): incompatible function arguments." in str(e.value)
 
 
-def test_allow_auto_batching_property():
+def test_allow_auto_batching_property(device):
     core = Core()
+    device_name = core.get_property(device, "FULL_DEVICE_NAME")
+
+    if device != "CPU" and "Intel" not in device_name:
+        pytest.skip("Properties are Intel(R) CPU specific")
```

Comment:
auto-batching is not CPU specific, it's generic functionality and we should not have any conditional logic here

Reply:
This test file was updated separately here: https://github.com/openvinotoolkit/openvino/pull/14684